diff -ur slurm-15.08.7/configure slurm-15.08.7.new/configure
--- slurm-15.08.7/configure	2016-01-20 21:56:09.000000000 +0000
+++ slurm-15.08.7.new/configure	2016-02-06 22:59:19.615761800 +0000
@@ -22858,8 +22858,8 @@
   if test "$x_ac_debug" = yes; then
     # you will most likely get a -O2 in you compile line, but the last option
     # is the only one that is looked at.
-    test "$GCC" = yes && CFLAGS="$CFLAGS -Wall -g -O0 -fno-strict-aliasing"
-    test "$GXX" = yes && CXXFLAGS="$CXXFLAGS -Wall -g -O0 -fno-strict-aliasing"
+    test "$GCC" = yes && CFLAGS="$CFLAGS -Wall -g -fno-strict-aliasing"
+    test "$GXX" = yes && CXXFLAGS="$CXXFLAGS -Wall -g -fno-strict-aliasing"
   fi
   { $as_echo "$as_me:${as_lineno-$LINENO}: result: ${x_ac_debug=no}" >&5
 $as_echo "${x_ac_debug=no}" >&6; }
